test/core/tsi/alts/crypt/aes_gcm_test.cc

```diff
@@ -803,8 +804,9 @@ static void gsec_test_create_random_aes_gcm_crypter(gsec_aead_crypter** crypter,
                                                     bool rekey) {
   uint8_t* key;
   gsec_test_random_array(&key, key_length);
-  gsec_aes_gcm_aead_crypter_create(key, key_length, nonce_length, tag_length,
-                                   rekey, crypter, nullptr);
+  gsec_aes_gcm_aead_crypter_create(key, key_length, /*copy_key=*/true,
```

Comment:
Could we run all of these tests twice, once with copy_key=true and once with copy_key=false?

Reply:
Done.

Reply:
It looks like the tests in this file are still only run with copy_key=true?

Reply:
Good point! I duplicated those tests taking copy_key and run them with either true or false in different test cases.